.circleci/config.yml

```diff
@@ -45,6 +46,38 @@ jobs:
       - run:
           name: Automated tests
           command: npm run test
+  analyze:
+    docker:
+      - image: sonarsource/sonar-scanner-cli:latest
+    steps:
+      - checkout
+      - run:
+          name: Get GitHub PullRequest Number
+          command: >-
+            if [[ -z "$CIRCLE_PR_NUMBER" ]]; then
+                if [[ -z "$CI_PULL_REQUEST" ]]; then
+                    URL="https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls?head=$CIRCLE_PROJECT_USERNAME:$CIRCLE_BRANCH"
+                    RESULT="`curl -X GET -u $GITHUB_ACCESS_TOKEN:x-oauth-basic $URL | jq ".[0].url"`"
+                    [[ "$RESULT" == 'null' ]] && CI_PULL_REQUEST='' || CI_PULL_REQUEST="${RESULT//\"}"
+                fi
+                CIRCLE_PR_NUMBER="$(basename "$CI_PULL_REQUEST")"
+            fi
+
+            echo 'export CIRCLE_PR_NUMBER=$CIRCLE_PR_NUMBER' >> $BASH_ENV
+      - run:
+          name: SonarQube - PullRequest Scan
+          command: >-
+            sonar-scanner
+            -X
+            -Dsonar.login=$SONAR_TOKEN
+            -Dsonar.host.url=https://sonarqube.platform.jalantechnologies.com
```

Comment:
Replace value here with `$SONAR_HOST_URL` (Please do check what's the env variable we are using here)